src/app/App.js

```diff
@@ -32,8 +34,14 @@ class App extends React.Component {
       .then(response => {
         this.setState({
           starCount: response.stargazers_count,
-        })
+        });
       })
+      .then(() => {
+        groupRepo.restoreGroup();
```

Comment:
if you use this pattern, this one must be async, or this scope will raises bugs later because of race condition with the block chain below.

we have `syncToPromise` from `infra/service/util` you can use it.